src/app/components/pages/MyProgress.tsx

```diff
@@ -51,8 +88,12 @@ const MyProgress = withRouter((props: MyProgressProps) => {
   const myAnsweredQuestionsByDate = useAppSelector(selectors.user.answeredQuestionsByDate);
   const userAnsweredQuestionsByDate = useAppSelector(selectors.teacher.userAnsweredQuestionsByDate);
 
+  const [subId, setSubId] = useState("correct");
```

Comment:
I don't know what is meant by `subId`, is this the right name for this? Although perhaps I'm just lacking the required domain knowledge, so feel free to ignore this comment if that's the case.

Reply:
I just saw that it's part of `QuestionProgressChartsProps`, which makes sense, so please ignore this!